conf/crontab.ugly

```diff
@@ -57,7 +57,7 @@ MAILTO=cron-!!(*= $site *)!!@mysociety.org
 # Check for any recurring data problems:
 47 4 * * * !!(*= $user *)!! run_management_command kenya_check_for_recurring_problems
 
-!!(* } elsif ($vhost eq 'nigeria.mzalendo.mysociety.org') { *)!!
+!!(* } elsif ($vhost eq 'www.shineyoureye.com') { *)!!
```

Comment:
Which is amazing as I got it correct two lines later :(

Fixed now. I will squash all this embarrassment down before I merge :|